src/Commands/DiffController.php

```
@@ -0,0 +1,104 @@
+
+            if ($stub === $current) {
+                $output[] = '  ' . ($stub ?? '');
+            } else {
+                if ($stub !== null) {
+                    $output[] = '- ' . $stub;
+                }
+
+                if ($current !== null) {
+                    $output[] = '+ ' . $current;
+                }
+            }
```

PHPMD (2.15.0)

[error] The method buildDiff uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)